src/DaySlot.jsx

```diff
@@ -150,11 +170,13 @@ let DaySlot = React.createClass({
         <div
           key={'evt_' + idx}
           style={{...xStyle, ...style}}
-          title={label + ': ' + title }
+          title={(typeof label ? label + ': ' : '') + title }
```

Comment:
ok that make sense, however you have no actual `typeof` check here, it should be `typeof label === 'string` right?